fonts.css

```diff
@@ -1,82 +1,422 @@
+
+	.character__item_little { font-size: 14px; }
+
+	.character__item_medium { font-size: 15px; }
+
+	.button__item_link-info { font-size: 11px; }
+}
+
+@media (max-width: 768px) {
+	.section-desc__title {
+		font-size: 68px;
+		line-height: 65px;
+		font-weight: 600;
+	}
+
+	.text__title {
+		font-size: 30px;
+		line-height: 40px;
+	}
+
+	.text__content {
+		font-size: 14px;
+		line-height: 18.4px;
+	}
+
+	.text__item_big {
+		font-size: 35px;
+		line-height: 40px;
+	}
+
+	.character__item_little { font-size: 20px; }
+
+	.character__item_small { font-size: 12px; }
+
+	.character__item_medium { font-size: 20px; }
+
+	.button__item_link,
+	.button__item_link-info { font-size: 16px; }
+}
+
+@media (max-width: 726px) {
+	.text__title { font-size: 28px; }
+
+	.text__content {
+		font-size: 12px;
+		line-height: 17.4px;
+	}
+
+	.character__item_medium { font-size: 15px; }
+
+	.button__item_link,
+	.button__item_link-info { font-size: 13px; }
+}
+
+@media (max-width: 650px) {
+	.text__title { font-size: 24px; }
+
+	.text__content {
+		font-size: 11px;
+		line-height: 15.4px;
+	}
+
+	.text__item_big { font-size: 30px; }
+
+	.character__item_medium { font-size: 15px; }
+
+	.character__item_little {
+		font-size: 16px;
+		line-height: 20px;
+	}
+	
+	.character__item_small {
+		font-size: 10px;
+		line-height: 13px;
+	}
+
+	.button__item_link,
+	.button__item_link-info { font-size: 11px; }
+}
+
+@media (max-width: 605px) {
+	.section-desc__title { font-size: 58px; }
+
+	.text__content { font-size: 10px; }
+
+	.character__item_font { font-size: 28px; }
+
+	.character__item_medium { font-size: 15px; }
+}
+
+@media (max-width: 560px) {
+	.text__title { font-size: 18px; }
+}
+
+@media (max-width: 465px) {
+	.section-desc__title { font-size: 53px; }
+
+	.character__item_medium { font-size: 14px; }
+
+	.character__item_font { font-size: 26px; }
+}
+
+@media (max-width: 400px) {
+	.text__title { font-size: 17px; }
+}
+
+@media (max-width: 375px) {
+	.text__title { font-size: 29px; }
+
+	.text__content {
+		font-size: 14px;
+		line-height: 19px;
+	}
+
+	.button__item_link { font-size: 13px; }
+}
+
+@media (max-width: 330px) {
+	.text__title { font-size: 27px; }
+}
+
+/* Адаптив третьей секции */	
+@media (max-width: 1410px) {
+	.star { font-size: 16px; }
+
+	.score { font-size: 38px; }
+
+	.score__item { font-size: 23px; }
+}
+
+@media (max-width: 1265px) {
+	.star {
+		font-weight: 600;
+		font-size: 12px;
+	}
+
+	.score { font-size: 30px; }
+
+	.score__item { font-size: 21px; }
+}
+
+@media (max-width: 1150px) {
+	.star {
+		font-weight: 600;
+		font-size: 12px;
+	}
+
+	.score {
+		font-size: 28px;
+		line-height: 27px;
+	}
+
+	.score__item { font-size: 19px; }
+}
+
+@media (max-width: 900px) {
+	.star { font-size: 11px; }
+
+	.star__title_add { font-size: 7px; }
+
+	.score {
+		font-size: 28px;
+		line-height: 27px;
+	}
+
+	.score__item { font-size: 19px; }
+}
+
+@media (max-width: 768px) {
+	.star {
+		font-weight: 700;
+		font-size: 17px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Project_page/pullRequest?prid=5536520)